lib/dry_twitter/registration/persist.rb

```diff
@@ -1,12 +1,19 @@
 require "dry/transaction/operation"
+require "dry_twitter/import"
 
 module DryTwitter
   module Registration
     class Persist
       include Dry::Transaction::Operation
+      include DryTwitter::Import["repositories.users"]
 
       def call(input)
```

Comment:
And again there is no spec for this :(

Reply:
If you do a spec for this you will find out something very very interesting about injection.